src/main/java/org/weasis/dicom/google/api/ui/dicomstore/DicomStoreSelector.java

```diff
@@ -335,6 +344,34 @@ public void updateTable(List<StudyView> studies) {
                 .flatMap(x -> (Optional<T>) x);
     }
 
+    /** Loads studies
```

Comment:
"Loads all DICOM studies that match the query"?

Reply:
done